monkey/monkey_island/cc/services/remote_run_aws.py

```diff
@@ -51,17 +51,24 @@ def update_aws_auth_params():
         Updates the AWS authentication parameters according to config
         :return: True if new params allow successful authentication. False otherwise
         """
+        AwsService.set_region(RemoteRunAwsService.aws_instance.region)
+
         access_key_id = ConfigService.get_config_value(['cnc', 'aws_config', 'aws_access_key_id'], False, True)
```

Comment:
Remove from config the values: 'aws_access_key_id, 'aws_account_id', 'aws_secret_access_key'.
Remove any reference to them. Notice the exporter for the securityhub uses them. Make sure you test the securityhub still works after all changes